### DISCUSSION TITLE
MPV and mplayer displays very different colors

### DISCUSSION THREAD

**Discussion by Bhaalspawn** [smplayer-dev/smplayer#445] [General]
Mplayer is much brighter.
This happens only on one PC, so there must be something wrong here.
i don't even know if it's a setting of smplayer or maybe something in the codecs

i tried to open the file with different video players and one is like mplayer while the other one is like MPV

MPV seems to be the right one to me but i can't be sure

any idea of where is the problem?

**Comment by smplayer-dev** [comment]
Try changing the video output in preferences -> general -> video.

**Reply by Bhaalspawn** [reply]
it didn't work
i think it's not something related only on smplayer. i don't know what it is but it's something on my PC that shows incorrct colors when using mplayer.
I tried to change amd drivers but it wasn't the problem
i uninstalled and resinstalled all the playyers i use but again i didn't solve the problem

i only know that on the other PC i don't have this problem

i can give info from 2 different PC, but i don't know where to search, what to show you
